src/getTime.js

```diff
@@ -19,6 +19,9 @@
  */
 function getTime(str) {
   // write code here
+  const timeRE = /\b(2[0-3]|[01]\d):[0-5]\d\b/;
```

Comment:
don't shorten names too aggresively. timeRegExp is easier to understand.

Reply:
Ok